base/applications/regedit/lang/ro-RO.rc

```diff
@@ -32,16 +32,16 @@ Opțiuni:\n\
   /R:user.dat    Locația fișierului user.dat urmează să fie modificată.\n\
                  Compatibil cu orice alt comutator. Ignorat.\n\
   /C             Importă conținutul unui fișier de regiștri.\n\
-  /D             Șterge o cheie de regiștrii specificată.\n\
-  /E             Exportă conținutul unei cheie de regiștri specificată într-un fișier.\n\
+  /D             Șterge o cheie de registru specificată.\n\
+  /E             Exportă conținutul unei chei de registru specificată într-un fișier.\n\
                  Dacă nu este nicio cheie specificată, toți regiștrii sunt exportați.\n\
   /S             Modul silențios. Nu sunt afișate mesaje.\n\
   /V             Lansează interfața grafică (GUI) în modul avansat. Ignorat.\n\
   /?             Afișează informații și părăsește programul.\n\
   [filename]     Locația fișierului ce conține informații despre regiștri ce urmează\n\
                  să fie importați. Când e folosit cu [/E], această opțiune specifică\n\
                  locația fișierului în care informațiile despre regiștri vor fi exportate.\n\
```

Comment:
Consider extending your change to also include the message of these three lines.